### PR TITLE
refactor ratings i18n strings to support ratings from different languages

### PR DESCRIPTION
**Pull request by psinghal20**
@ragesoss I have edited the i18n strings. Please take a look at them and mention any changes, if required.

### CODE REVIEW

**Comment by ragesoss** [comment]
Can you show how this will look in a language other than English?

**Comment by psinghal20** [comment]
@ragesoss 
Article ratings for Hungarian language(hu):
screenshot from 2018-06-20 05-03-40

The class is displayed via interpolation and remaining message is from the language locale.

**Comment by ragesoss** [comment]
Cool! Needs a space between the interpolated class and the rest of the string?

**Comment by psinghal20** [comment]
@ragesoss done!

File: config/locales/en.yml [hunk 1]
```diff
@@ -146,15 +146,15 @@ en:
...
-      a: A-class — very developed article that may be too mature an article for a new editor's main project
-      ga: Good article — a well-developed article that may be too mature an article for a new editor's main project
-      b: B-class — a moderately developed article
-      c: C-class — an intermediate article with room for improvement
-      start: Start-class — a preliminary article with plenty of room for improvement
-      stub: Stub-class — a short beginning of an article that needs more content
-      list: List-class — a list rather than a conventional article
+      fa: "%{class} (Featured article) — Wikipedia's highest rating, probably too mature an article for a new editor's main project"
+      fl: "%{class} (Featured list) — Wikipedia's highest rating for lists"
+      a: "%{class} - class — very developed article that may be too mature an article for a new editor's main project"
```
Comment (ragesoss) [inline config/locales/en.yml]: the ones with `-class` don't need spaces, just the ones with parentheses.

**Pull request merged**